scripts/install.sh

```diff
@@ -110,7 +110,7 @@ check_flag() {
 
 GO_VERSION="1.21.7"
 NODE_VERSION="20.9.0"
-PNPM_VERSION="8.8.0"
+PNPM_VERSION="9.2.0"
```

Comment:
could this cause the pnpm lock changes?

Reply:
No. pnpm-lock was updated when I called pnpm update.

Reply:
ah I see